doc/debugging.md

```diff
@@ -105,8 +118,8 @@ sudo tio -b 115200 -m INLCRNL /dev/ttyACM0
 4. Ground target to host
     - Connect USB cable from USB-C port on target to host
 5. Connect target to host
```

Comment:
They will also need to flash firmware to the trinket which takes I2C writes to 0x76 and prints them to the console

Reply:
This can be done in another commit